### DISCUSSION TITLE
can i use pem & crt file with your websocketclient ?

### DISCUSSION THREAD

**Discussion by nguyenngocanh94** [jchristn/WatsonWebsocket#63] [General]
i have both pem and crt file to create pfx file and use it to connect to ws server.
but i didn't see any where setup it? can you clear me that?

**Comment by jchristn** [comment]
Hi, you should be able to use openssl to convert to PFX: https://stackoverflow.com/questions/808669/convert-a-cert-pem-certificate-to-a-pfx-certificate

You would then install the PFX into your OS and use ```netsh``` to bind the cert by thumbprint to a port: https://github.com/jchristn/WatsonWebsocket/wiki/Using-SSL-on-Windows
